help_to_heat/templates/frontdoor/base.html

```diff
@@ -29,33 +29,37 @@
 </head>
 
 <body class="govuk-template__body" id="i-dot-ai">
-  {% if request.COOKIES['cookiesAccepted'] != "True"%}
-  <div id="cookie-banner" class="govuk-cookie-banner" data-nosnippet role="region"
-    aria-label="Cookies on Check eligibility for Great British Insulation Scheme">
-    <div class="govuk-cookie-banner__message govuk-width-container">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-          <h2 class="govuk-cookie-banner__heading govuk-heading-m">Cookies we are using on our service</h2>
-          <div class="govuk-cookie-banner__content">
-            <p class="govuk-body">We will use necessary cookies when you use this service to help you move around the
-              site.</p>
-            <p class="govuk-body">We will also use analytic cookies using Google Analytics to tell us how you got to our
-              service, the pages you visit, and
-              how long you spent on them. Analytics cookies help us to improve our service. We do not allow Google to
-              use or share the
-              data about how you use this site and the data is stored anonymously.</p>
+  {% if not SUPPRESS_COOKIE_BANNER %}
```

Comment:
I believe this can just be merged into the conditional below, to avoid needing two `if`s?

Reply:
Ah yes! It complains about formatting but still works 😂
![image](https://github.com/UKGovernmentBEIS/help-to-heat-GBIS/assets/104062694/5c2b7e49-46b4-4bcf-9057-4872a66de7ea)